.github/workflows/build.yaml

```diff
@@ -9,6 +9,10 @@ on:
     branches:
       - main
 
+  schedule:
```

Comment:
I combined this into the existing Build workflow; the consequence is that we'll rebuild from source daily, but I think that's a good thing to avoid bitrot. The scan runs on every pull request but is non-blocking